_posts/2025-04-17-lxqt-menu-data-2.2.0.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Release lxqt-menu-data 2.2.0
+slug: release-lxqt-menu-data-2-2-0
+date: 2025-04-15 0:05
+promoted: false
+categories: release
+---
+
+The LXQt team announces the release of lxqt-menu-data 2.2.0.
+
+Changes:
+
+ * Updated translations and bumped the version.
+
+ <br/>
+
+
+The release can be downloaded from [Github](https://github.com/lxqt/lxqt-menu-data/releases).
+
+<br/>
+The complete list of changes is available in our CHANGELOG file.
```

Comment:
Not sure if intended, but this is different from the others which have:

`A full list of changes is in the CHANGELOG file.`